app/mailers/notifier.rb

```diff
@@ -0,0 +1,9 @@
+class Notifier < ApplicationMailer
+  default from: 'sunny@treeni.com',
+          return_path: 'sunny@treeni.com'
```

Comment:
replace the email with a more relevant sender. selfassessment@unglobalcompact.org or something.